src/main/java/cz/mzk/fedora/solrcorrection/RemoveVcSolr.java

```diff
@@ -0,0 +1,22 @@
+package cz.mzk.fedora.solrcorrection;
+
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrServerException;
+
+import java.io.IOException;
+
+public class RemoveVcSolr {
+
+    public static void main(String[] args) throws SolrServerException, IOException {
+        String solrHost = System.getenv("SH");
```

Comment:
kdyz pristi nekde nekam budes davat prilis strucne env promenne, tak to muze zpusobit neporozumeni ze strany kolegu
treba ja bych nepochopil nekde promennou "SH", ale s napriklad "SOLR_HOST" je vsechno jasne)